### ISSUE TITLE
RM not recognized on Windows 

### ISSUE THREAD

**Issue by jordanforbes** [cifkao/html-midi-player#58]
running npm start on most windows terminals doesn't work because 'rm' is replaced with 'del' 
That said you can get around this by using git bash, but I think it's definitely something to make note of.

**Comment by cifkao** [comment]
Good point. Please feel free to make a pull request if you can think of a cross-platform way to write it.

**Comment by cifkao** [comment]
Maybe [via `rimraf`](https://stackoverflow.com/a/53920043) but it would need to be added to dev dependencies.
